state_machine/applications/system-check/tests/imu_test.py

```diff
@@ -0,0 +1,262 @@
+"""
+Python system check script for PyCubed satellite board
+PyCubed Mini mainboard-v02 for Pocketqube Mission
+IMU Sensor Test
+* Author(s): Yashika Batra
+"""
+
+import time
+from ulab import numpy
+from lib import pycubed as cubesat
+
+
+wait_time = 2
+norm = numpy.linalg.norm
+
+
+def stationary_user_test():
+    """
+    All user interaction for the stationary test happens in this function
```

Comment:
This docstring doesn't really tell me much about what this function does, could you please describe the behavior?